Adding a gradio demo of SVD to be run locally

yvrjsharma opened the pull request:
added gradio_app.py, updated requirements/pt2.txt and readme.md for relevant libraries and information respectively

akx commented on requirements/pt2.txt:
```diff
@@ -37,3 +37,6 @@ wandb>=0.15.6
 wheel>=0.41.0
 xformers>=0.0.20
+fire
+uuid
```
uuid is a standard library, it doesn't need to be a requirement.

akx commented on scripts/demo/gradio_app.py:
```diff
@@ -0,0 +1,322 @@
+import numpy as np
+import torch
+from einops import rearrange, repeat
+from fire import Fire
```
This import (and the dependency) is unused.

timudk commented:
Awesome thank you so much @yvrjsharma!! This is amazing and so fast!

Before merging I have a couple of requests:
- add gradio to reqs (and remove fire, uuid as @akx suggested)
- only download ckpt if not already present (it redownloaded it for me)
- it would be nice to add options for all models (instead of only svd-xt), it's optional but if not we should rename the file
- fps_id should be an int (but slider is float)
- a lot of code seems to be copy & pasted from `simple_video_sample`, can we instead just import functions we need?

Sumith1896 commented:
@yvrjsharma thanks for the PR! Could you please address @timudk's comments from above and let us know? One of us will have look (cc @AK391)

yvrjsharma commented:
Hi, @timudk @Sumith1896, sincere apologies I couldn't come back to this thread any earlier due to various reasons. 
I have addressed all the comments from above. Thanks for the detailed comments and for following up on this! Let me know if this PR need anything else from my side.

Sumith1896 reviewed (approved):
tried locally and works, thanks for the PR @yvrjsharma!
